great_expectations/core/usage_statistics/anonymizers/__init__.py

```diff
@@ -1,9 +0,0 @@
-from .batch_anonymizer import BatchAnonymizer
-from .batch_request_anonymizer import BatchRequestAnonymizer
-from .checkpoint_run_anonymizer import CheckpointRunAnonymizer
-from .data_docs_site_anonymizer import DataDocsSiteAnonymizer
-from .datasource_anonymizer import DatasourceAnonymizer
-from .execution_engine_anonymizer import ExecutionEngineAnonymizer
-from .expectation_suite_anonymizer import ExpectationSuiteAnonymizer
-from .store_anonymizer import StoreAnonymizer
-from .validation_operator_anonymizer import ValidationOperatorAnonymizer
```

Comment:
See the Loom video noted in the description for more context here - with this `__init__.py` populated with imports, any import statements that touches the anonymizers causes all of them to be imported together.

Rather than that, we want to only import each anonymizer when it is appropriate to do so. The issue manifests through the `CheckpointRunAnonymizer` but the general idea applies to the rest of the classes here.